src/architecture/clients.md

```diff
@@ -38,12 +39,15 @@ There are two typical use cases for the light client:
 In this and next section all functions indicated in *italics* are the functions
 implemented in Exonum light client.
 
-1. Frontend triggered by an event (for example, a button click handler)
-  decides to create a new transaction
-2. The transaction data is stored in JSON format. Then the data is
-  *converted to the Exonum binary format* and *digitally signed*
-3. The generated transaction (JSON data + digital signature) is sent to the
-  a full node via an HTTP POST request
+![Send data to the blockchain](../images/send-data.png)
+
+1. Frontend triggered by an event (for example, a button click handler) decides to create a new transaction.
```

Comment:
I suggest to use "Frontend application" and "light client library" terms. The first is to match the figure, and the second is because without this clarification, it could be difficult to separate the frontend application and the light client.